proposal/color-4-new-spaces.changes.md

```diff
@@ -1,3 +1,7 @@
+## Draft 1.14
+
+* Update the definition of powerless for HWB to match the latest CSS spec.
```

Comment:
Nit: Is there a changelog on the latest CSS spec? If so, it would be nice to link to that changelog here